src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
```

Comment:
align entire csproj with the other projects - take a look to \src\ResourceManagement\Resource\ResourceManagement\ResourceManagement.csproj"